implementations/micrometer-registry-dynatrace2/src/main/java/io/micrometer/dynatrace2/Streams.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020 VMware, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.dynatrace2;
+
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+class Streams {
```

Comment:
Why is this needed, would not calling `StreamSupport.stream` be ok (the rest of the codebase does that)?
I think if we can agree that dropping the parallel parameter has some value that worth the change, we can do it everywhere but I would do it in a separate PR.